Domain/Projects/Phase.cs

```diff
@@ -4,14 +4,21 @@ namespace Domain.Projects
 {
     public class Phase
     {
+        public Phase()
+        {
+            IsCurrentPhase = false;
+        }
+
         //Added by NG
         public int Id { get; set; }
-        public Project Project { get; set; }       
+        public Project Project { get; set; }
         public Module Module { get; set; }
         public string Description { get; set; }
         public DateTime StartDate { get; set; }
         public DateTime EndDate { get; set; }
 
+        public bool IsCurrentPhase { get; set; }
```

Comment:
Ik snap het nut van deze property maar is dit de responsibility van het domein om dit bij te houden. Zo ja moet deze boolean worden gepersisteerd?